packages/web/src/store/slices/home.ts

```diff
@@ -3,7 +3,11 @@ import { client } from '../../services/apiClient';
 import { RootState } from '../';
 
 const parseWebsite = createAsyncThunk('home/submitWebsite', async (url: string) => {
-  await client.mutation('requestParseWebsite', url);
+  if (!url.startsWith('http://') && !url.startsWith('https://')) {
```

Comment:
nit: the URL validation and formatting should probably be inside a component logic